akka-docs/src/main/paradox/stream/operators/Source-or-Flow/dropWhile.md

```diff
@@ -16,6 +16,16 @@ Drop elements as long as a predicate function return true for the element
 
 Drop elements as long as a predicate function return true for the element
 
+## Example
+
+Given a `Source` of ordered numbers we can drop all the negative ones with the `dropWhile` operator: 
```

Comment:
Please describe the distinction to `filter`: `dropWhile` will emit all elements after the expression returns false for an element.

Reply:
Added additional explanations